tests/simulator/sim_test.go

```diff
@@ -102,6 +114,7 @@ func fullAppSimulation(tb testing.TB, is_testing bool) {
 		initFns,
 		osmosis.SimulationManager().Actions(config.Seed, osmosis.AppCodec()), // Run all registered operations
 		config,
+		sqlDB,
```

Comment:
Wait why are we passing in a new db in the simulator instantiation? The simulator itself should handle creating and cleaning up the entire db

Reply:
Sorry I misunderstood your original comment! I had decided to do it at the highest level to ensure I don't close the DB prematurely but on further review it should be a non issue if I just defer this within the simulateFromSeed function itself